chart/hyrax/templates/configmap-env.yaml

```
@@ -19,15 +19,20 @@ data:
   HYRAX_CACHE_PATH: "/app/samvera/file_cache"
   HYRAX_UPLOAD_PATH: "/app/samvera/uploads"
   {{- end }}
+  {{- if .Values.loadHyraxEngineSeed }}
+  LOAD_HYRAX_ENGINE_SEED: "true"
```

the extra layer of boolean interpretation added by Go makes me slightly nervous. maybe:

```suggestion
  LOAD_HYRAX_ENGINE_SEED: {{  .Values.loadHyraxEngineSeed | default 0 }}
```

---

Yeah, I'm not crazy about it either. This seems like a good alternative, thanks!